packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapBuilder.java

```diff
@@ -36,7 +35,7 @@ GoogleMapController build(
       BinaryMessenger binaryMessenger,
       Application application,
       Lifecycle lifecycle,
-      PluginRegistry.Registrar registrar,
+      @SuppressWarnings("deprecation") io.flutter.plugin.common.PluginRegistry.Registrar registrar,
```

Comment:
@cyanglaz or @blasten what's the plan for this longer term? I remember something funky about this plugin and the v2 embedding but not specifics.

Reply:
@dnfield it's fixed